methods/basesim.R

```diff
@@ -0,0 +1,205 @@
+# required packages
+
+library(magrittr)
+library(escalation)
+library(ggplot2)
+
+# lists
+
+n_sims <- list()
+
+model <- list()
+sims <- list()
+
+selection <- list()
+selection_df <- list()
+selection_tab <- list()
+
+treated <- list()
+treatedpct <- list()
+treatedpct_df <- list()
+treatment_tab <- list()
+
+dist_accuracy <- list()
+mean_accuracy <- list()
+
+dist_overdose <- list()
+mean_overdose <- list()
+
+dist_length <- list()
+mean_length <- list()
+
+# variables needed (UI)
```

Comment:
It is fine for now to include the dummy values for the required variables. In the next iteration, you can remove this step and add a new function to read in variables for each method as appropriate